lab_1/start.py

```diff
@@ -2,17 +2,45 @@
 Concordance implementation starter
 """
 
-from main import read_from_file
 import os
+import main

```

Comment:
Usually, the import statement goes like this:
`from main import func1, func2, func4 ...` unless you want to use all functions from the module.

Reply:
I want to use all functions from the main module.